docs/io/unified.rst

```diff
@@ -36,12 +36,25 @@ The example below shows how to read a table in the specialized DAOphot format an
 it back to FITS format. Notice that FITS is a format where the interface recognizes the
 format automatically from the file name, so the ``format`` argument is not needed.
 
-.. doctest-skip::
+.. testsetup::
```

Comment:
I think I'd make writing the test file part of the documentation - it is always good if the user can copy & paste examples.